tests/dummy/config/ember-intl.js

```diff
@@ -0,0 +1,64 @@
+/*jshint node:true*/
```

Comment:
Since the config file is no longer needed (and still is not loaded as far as i can tell) you could also remove it. I inspected the service with Ember Inspector and the locales are only set from the route. This is ignored:
```js
locales: ["fr-FR", "en-us"],
```

Reply:
Is the config file working for you? As mentioned i don't think it is loaded locally.

Reply:
Removed